### ISSUE TITLE
Bing Search

### ISSUE THREAD

**Issue by AlNotAlbert** [infokiller/web-search-navigator#540]
### Describe the problem you're trying to solve

The new bing is amazing to use, it would be nice to have keyboard navigation

### Describe the solution you'd like

extending the keybinds to work on the new bing search page with a button to switch to chat

### Describe alternatives you've considered

switching browsers

**Comment by infokiller** [comment]
Ack, when I have free time I'll check how difficult it is to support it

**Comment by MustCodeAl** [comment]
any updates?

**Comment by infokiller** [comment]
No, and I probably won't have time to work on that in the near future
